index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+    <main>
+      <div class="container">
+        <div class="name">
+          <ul class="na">
+            <li>Emma Ngufor</li>
+          </ul>
+        </div>
+  
+        <div class="name1">
+          <ul class="name2">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><img src="image/Union.png" alt="an envelope"></li>
+          </ul>
+        </div>
+      </div>
+  
+      <h1>Hey There.<br> I'm Emmanuella</h1>
+      <h4> I'am a Software Developer</h4>
+      <p>I can help you build a product, feature or a website look through some 
+        of my work and experiment if you like what you see and have a project you need 
+        coded, don't hestiate to contact me. 
+      </p>
+  
+      <div class="content-image">
+        <img src="image/Illustration-header@2x.png" alt="a beautiful design">
+      </div>
+  
+      <aside>
+        <ul class="icons">
+          <li><img src="image/github.svg" alt="github icon"></li>
+          <li><img src="image/Linkedin-icon.svg" alt="Linkedin-icon"></li>
+          <li><img src="image/angel-list.svg" alt="angel list icon"></li>
+          <li><img src="image/twitter.svg" alt="twitter icon"></li>
+          <li><img src="image/medium.svg" alt="medium icon"></li>
+        </ul>
+      </aside>
```

Comment:
Kindly Wrap this block in a section tag since later on you will add more sections to your project 😀

- _A section represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info._